.github/workflows/e2e-nm-angular-workflow.yml

```diff
@@ -48,12 +48,11 @@ jobs:
 
     - name: 'Running the integration test'
       run: |
-        source scripts/e2e-setup-ci.sh;
+        source scripts/e2e-setup-ci.sh nm
 
         yarn dlx -p @angular/cli@next ng new berry-angular --interactive=false
 
         cd berry-angular
-        echo 'nodeLinker: node-modules' >> .yarnrc.yml
         echo 'enableGlobalCache: true' >> .yarnrc.yml
         echo 'compressionLevel: 0' >> .yarnrc.yml
```

Comment:
Should we set this earlier in the environment so the `dlx` and `add` commands can share the cache?